SK.Application/TestValues/Commands/Edit/EditTestValueCommand.cs

```diff
@@ -0,0 +1,45 @@
+﻿using MediatR;
+using SK.Application.Common.Exceptions;
+using SK.Application.Common.Interfaces;
+using SK.Domain.Entities;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SK.Application.TestValues.Commands.Edit
+{
+    public class EditTestValueCommand
+    {
+        public class Command : IRequest
+        {
+            public int Id { get; set; }
+            public string Name { get; set; }
+        }
+
+        public class Handler : IRequestHandler<Command>
+        {
+            private readonly IApplicationDbContext _context;
+
+            public Handler(IApplicationDbContext context)
+            {
+                _context = context;
+            }
+
+            public async Task<Unit> Handle(Command request, CancellationToken cancellationToken)
+            {
+                var testValue = await _context.TestValues.FindAsync(request.Id);
+
+                if (testValue == null)
+                {
+                    throw new NotFoundException(nameof(TestValue), request.Id);
+                }
+
+                testValue.Id = request.Id;
+                testValue.Name = request.Name;
```

Comment:
Test value is just temporary entity which allows me to prepare clean project template and test db operations. After receiving specification from client it will be deleted and new entities based on project documentation will be much more understandable.